RELEASES.md

```diff
@@ -4,6 +4,24 @@ Go [here](https://github.com/PSLmodels/Tax-Calculator/pulls?q=is%3Apr+is%3Aclose
 for a complete commit history.
 
 
+2020-02-28 Release 2.8.0
+------------------------
+(last merged pull request is
+[#2400](https://github.com/PSLmodels/Tax-Calculator/pull/2400))
```

Comment:
I think this should be #2399

Reply:
I put the PR of the very last merged PR, which is this one.